### PR TITLE
channel archiver

### PR DESCRIPTION
**Pull request by licode**

### CODE REVIEW

File: recipes/channelarchiver/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  name: channelarchiver
+  version: {{ version }}
+
+source:
+  fn: channelarchiver-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/c/channelarchiver/channelarchiver-{{ version }}.tar.gz
+  md5: ec0c0f80af8ebfc11ec239b299182592
+
+build:
+    number: 0
```
Comment (ocefpaf) [inline recipes/channelarchiver/meta.yaml]: @licode if you only want Linux you can do:

`skip: True  # [not linux]`

Comment (licode) [inline recipes/channelarchiver/meta.yaml]: thanks.

**Comment by ocefpaf** [comment]
:tada: First recipe !!! :tada:

**Comment by licode** [comment]
appreciate that. thanks!

**Pull request merged**